### ISSUE TITLE
LWT plugin no longer working with new website as of 10/02/2015

### ISSUE THREAD

**Issue by CommanderBubble** [garysharp/Disco-LWTPlugin#1]
When trying to log a warranty job or non-warranty job against LWT from disco, we are now getting an internal error, and a stack trace.

When probing it for details, it references "Disco\Source Repository\Disco-LWTPlugin\LWT.DiscoPlugin\Internal\LWTJobs.cs:line 118"

Checking this on Github shows that the streamreader is unable to parse the string. the information it appears to be trying to parse is the website "http://www.lwt.com.au/warranty.asp" which no longer resolves on lwt's website.

granted this has only started today, but do you think it will take long to come up with a fix?

Below is the a link to the stack trace since if i paste it here it renders the xml document (thought that shows the error).

https://www.dropbox.com/s/zd7tcj00ppyj0za/stack_trace.txt?dl=0

**Comment by garysharp** [comment]
Thanks for the detailed report.

I've contacted LWT and they say the issue is now resolved (as of about [9:15am AEDT](http://www.timeanddate.com/worldclock/fixedtime.html?iso=20150209T2215)). They explained that some components were forgotten during a recent website migration and have now been deployed.

I've received several reports that indicate the confirmation email is still not being sent. This has also been reported to LWT. LWT confirmed that they _are_ receiving jobs even though the confirmation email is not sent - the presence of the Warranty Reference indicates jobs have been processed:
![lwtwarrantyreference](https://cloud.githubusercontent.com/assets/1687386/6119490/7ff2c612-b11d-11e4-9d8c-fabd2272f77b.png)
